### PR TITLE
[AMR-85] Update Updater

### PR DESCRIPTION
**Pull request by N0chteil**

### CODE REVIEW

File: src/browser/index.html [hunk 1]
```diff
@@ -131,6 +131,72 @@ <h3 data-i18n="settings.category.general">
...
+                                <div class="setting_main">
+                                    <label data-for="config_autoUpdates">
+                                        Automatic Updates
+                                    </label>
+                                    <label class="cfgSwitch cfg_loading">
+                                        <input
+                                            type="checkbox"
+                                            id="config_autoUpdates"
+                                        />
+                                        <i></i>
```
Comment (MarconiGRF) [inline src/browser/index.html:157]: Is this element serving any purpose? If not, please remove it.

File: src/browser/index.html [hunk 2]
```diff
@@ -131,6 +131,72 @@ <h3 data-i18n="settings.category.general">
...
+                                <div class="setting_main">
+                                    <label data-for="config_betaUpdates">
+                                        Beta Updates
+                                    </label>
+                                    <label class="cfgSwitch cfg_loading">
+                                        <input
+                                            type="checkbox"
+                                            id="config_betaUpdates"
+                                        />
+                                        <i></i>
```
Comment (MarconiGRF) [inline src/browser/index.html:172]: Last comment applies here too.

**Review by MarconiGRF — CHANGES_REQUESTED** [review]

File: src/browser/index.html (hunk 1 above)
Comment (N0chteil) [reply src/browser/index.html:157]: Yes, it's needed for the switch style.

File: src/browser/index.html (hunk 2 above)
Comment (N0chteil) [reply src/browser/index.html:172]: Yes, it's needed for the switch style.

**Review by MarconiGRF — APPROVED** [review]

**Pull request merged**